dotCMS/src/integration-test/java/com/dotmarketing/db/listeners/CommitAPITest.java

```diff
@@ -0,0 +1,114 @@
+package com.dotmarketing.db.listeners;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.Map;
+import java.util.UUID;
+import org.junit.Test;
+import com.dotcms.repackage.net.sf.hibernate.HibernateException;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotHibernateException;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.UUIDGenerator;
+import com.tngtech.java.junit.dataprovider.DataProvider;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+
+public class CommitAPITest {
+
+    @DataProvider
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit 4 tests that execute tests should use the @Test annotation, JUnit 5 tests should use @Test, @RepeatedTest, @TestFactory, @TestTemplate or @ParameterizedTest](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5639225)